src/app/components/App/AppContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import * as firebase from "firebase";
+import { connect } from 'react-redux';
+import { setAuthState } from '../../redux/actions.js';
+import App from './App.js';
+
+var config = {
```

Comment:
Is it actually an issue to have this public? I could have sworn that someone said the apiKey wasn't information that needed to be kept secret.

Source: http://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public